lib/events.js

```diff
@@ -82,6 +80,7 @@ const {
   validateBoolean,
   validateFunction,
   validateString,
+  validateNumber,
```

Comment:
nit: ASCII order
```suggestion
  validateNumber,
  validateFunction,
  validateString,
```

Reply:
I think you intended below. (F => N => S) Is this right?
```
  validateFunction,
  validateNumber,
  validateString,
```